packages/manager/modules/nasha/src/dashboard/partition/snapshots/snapshots.controller.js

```diff
@@ -28,6 +29,8 @@ export default class NashaDashboardPartitionSnapshotsController {
     this.customSnapshotNamePattern = CUSTOM_SNAPSHOT_NAME_PATTERN;
     this.customSnapshotNamePrefix = CUSTOM_SNAPSHOT_NAME_PREFIX;
     this.customSnapshotNameSeparator = CUSTOM_SNAPSHOT_NAME_SEPARATOR;
+
+    this.MAX_CUSTOM_SNAPSHOT = MAX_CUSTOM_SNAPSHOT;
```

Comment:
Suggestion: Can we move the calculation {{ count }}/{{ max }}to controller ? 
 this.maximum = this.customSnapshots.length / MAX_CUSTOM_SNAPSHOT;

Reply:
This is not a calculation but a display of information. Would you prefer I use this.$translate.instant()?